src/makeRobotAccountant.js

```diff
@@ -7,6 +7,25 @@
 
 function makeRobotAccountant() {
   // write code here
+  let count = 0;
+
+  return function(a) {
+    return function(b) {
+      count++;
+
+      if (count <= 3) {
+        return a + b;
+      }
+
+      if (count >= 4 && (count % 2 === 0)) {
```

Comment:
```suggestion
      if (count % 2 === 0) {
```
Ты уже предыдущим if-ом проверил что coutn >= 4